src/main/resources/drop.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE orders DROP CONSTRAINT FK_orders_customers;
```

Comment:
Why you DROP Constraint exlicitly instead of implicit DROP with Table?

Reply:
I didn't know about that. Thanks, I'll research it.